src/libs/EmojiUtils.js

```diff
@@ -255,7 +255,7 @@ function replaceEmojis(text, isSmallScreenWidth = false, preferredSkinTone = CON
 
     // Add all replaced emojis to the frequently used emojis list
     if (!_.isEmpty(emojis)) {
-        addToFrequentlyUsedEmojis(emojis);
+        getFrequentlyUsedEmojis(emojis);
```

Comment:
Should this update the frequently used emojis user?

Reply:
Thanks for pointing that out. I've missed this function usage before.

I think that we also need to move this line out of `replaceEmojis` function and code a separate logic to handle the API update. Please take a look at my new code.